src/features/Treege/components/Forms/FormTreeCardMutation/FormTreeCardMutation.tsx

```diff
@@ -87,20 +87,26 @@ const FormTreeCardMutation = ({ onClose }: FormTreeCardMutationProps) => {
     handleChangeIsDecisionField,
     handleChangeOptionLabel,
     handleDeleteValue,
+    handleDeleteParam,
     handleChangeOptionValue,
     handleChangeMessage,
     handleSubmit,
     handleAddValue,
+    handleAddParams,
     handleChangeLabel,
     handleChangeRepeatable,
     handleChangeHiddenValue,
     handleChangeTag,
     handleChangeSearchKey,
     handleChangeUrl,
-    url,
-    searchKey,
+    handleChangeParam,
+    handleChangePath,
+    route,
   } = useFormTreeCardMutation();
 
+  const { searchKey, url, pathKey, params } = route || {};
+  const { object: routeObject, label: routeLabel, value: routeValue, image: routeImage } = pathKey || {};
```

Comment:
Il y a une erreur quand tu commence à saisir un champ de `pathKey` car tu passe d'un champs `uncontrolled` à `controlled`. Pour éviter ce genre d'erreur il faut initialiser la valeur de ta state car si value est `undefined` ton champ est considérer comme `uncontrolled`.

![Capture d’écran 2024-02-07 à 15 08 57](https://github.com/Tracktor/treege/assets/108873902/920f56dd-a1be-45f4-8c28-903ff0e7678f)


Pour le fix (met toi à jour avant sur ta branch car j'ai merge master dessus et ajouter la new version) : 

```suggestion
  const { object: routeObject = "", label: routeLabel = "", value: routeValue = "", image: routeImage = "" } = pathKey || {};
```

Reply:
j'ai fais la modif !